charts/trino/templates/configmap-coordinator.yaml

```
@@ -63,7 +63,7 @@ data:
     http-server.https.port={{ .Values.server.config.https.port }}
     http-server.https.keystore.path={{ .Values.server.config.https.keystore.path }}
   {{- end }}
-  {{ .Values.server.coordinatorExtraConfig | indent 4 }}
+{{ .Values.server.coordinatorExtraConfig | indent 4 }}
```

You could try this to preserve alignment and still have correct indentation.
```suggestion
  {{- .Values.server.coordinatorExtraConfig | indent 4 }}
```

---

This results in

```
  config.properties: |
    coordinator=true
    node-scheduler.include-coordinator=false
    http-server.http.port=8080
    query.max-memory=4GB
    query.max-memory-per-node=1GB
    discovery.uri=http://localhost:8080    query.max-run-time=3m
    - query.max-scan-physical-bytes=1GB
```